libs/antsi/src/font.rs

```diff
@@ -106,25 +109,57 @@ pub enum Blinking {
 }
 
 #[derive(Debug, Copy, Clone, Eq, PartialEq)]
+#[cfg(feature = "script")]
 #[non_exhaustive]
 pub enum FontScript {
     Sub,
     Super,
 }
 
+macro_rules! impl_builder {
+    (
+        $(#[$meta_set:meta])*
+        $set:ident($set_var:ident : $set_ty:ty);
+        $(#[$meta_with:meta])*
+        $with:ident($with_var:ident : $with_ty:ty);
+        $(#[$meta_getter:meta])*
+        $get:ident() -> $ret:ty
+    ) => {
+        $(#[$meta_set])*
+        pub fn $set(&mut self, $set_var: $set_ty) -> &mut Self {
+            self.$set_var = Some($set_var);
+            self
+        }
```

Comment:
There may be situations where setters are useful, but we probably don't want to provide two different builder patterns at the same time. Also, it's unintuitive, that the setters implies `Some(value)`, so it's not possible to `unset()`.

Reply:
macro removed in [9b2891a](https://github.com/hashintel/hash/pull/2248/commits/9b2891afbf6365b877c748d25cecf78e7f5ef90e)